lane.py

```diff
@@ -669,8 +690,8 @@ class FeaturedFacets(FacetsWithEntryPoint):
 
     DETERMINISTIC = object()
 
-    def __init__(self, minimum_featured_quality, uses_customlists=False,
```

Comment:
`uses_customlists` is gone because `FeaturedFacets` no longer needs to know whether a `WorkList` uses custom lists. The Elasticsearch scoring function will work the same way whether or not a work is on any lists, and the code to calculate the 'quality tier' field through SQL has been removed.